README.md

```diff
@@ -1,7 +1,25 @@
 # Django Girls Tutorial
 
-This is the source code repository for the Django Girls Tutorial. Django Girls Tutorial is used on [tutorial.djangogirls.org](http://tutorial.djangogirls.org) so if you want to read it, please go there. If you want to contribute please do go further with this file.
+~ 🖥 Source Code 🖥 ~
 
+**NOT** intended for reading here. To read go to: [📖 tutorial.djangogirls.org 📖](http://tutorial.djangogirls.org)
```

Comment:
I only see this now, but it was a good chance to also use `https://` in the link.

Reply:
Thank you! I've created patch for this: https://github.com/DjangoGirls/tutorial/pull/1804#issuecomment-1749736296